frontend/src/app/modules/matches/matches-selectors.js

```diff
@@ -52,6 +51,6 @@ export const getStatisticsForPlayer = createSelector(
 
 export const getRatingHistoryGraphForPlayer = createSelector(
   getLastMatchesForPlayer,
-  getPlayer,
+  (state, playerId) => state.players.find(player => player.id === playerId),
```

Comment:
I had to change this due to (probably) circular dependency in selectors between `matches-selectors` and `players-selectors`:
```
Error: Selector creators expect all input-selectors to be functions, instead received the following types: [function, undefined]
```
An ideas about alternative solution are welcome!

Reply:
usually the solution for A <-> B is to add C :) so A <- C, B <- C :) so I guess some selectors could be put outside of matches/players ones

Reply:
I thought about it for a bit and decided that conceptually, there is no need for third group of selectors as they are all still selectors of players and matches. I tried to reorganize and refactor selectors and computations but it is hard to decide when a thing working with both players and matches should be labeled as `players-selector` and when as `matches-selector`. I used as a rule of a thumb some similaties (such as all `getXXXForPlayer` selectors are together) and general functionalities of things (like `computeKingStreakDuration` seems more of players-computation rather then matches one).

What do you think about the current design and what would you improve/change @littlewhywhat ?

Reply:
if it works and there is no unnecessary code duplication (it can be accepted sometimes though), I am fine with it :) I was mainly answering your question) I don't mind that much about organizing things on a such low level... I think in this case, one (or at least me) would use IDE tools to navigate through those function anyway. And there are probably hundreds of way to organize these functions and none of them is probably perfect :man_shrugging: so if it makes sense to you it's good :+1: